<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>AddDialog</class>
    <widget class="QDialog" name="AddDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>300</width>
                <height>200</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Добавить / Редактировать</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
                <widget class="QLabel" name="labelOrderNumber">
                    <property name="text">
                        <string>Номер заказа</string>
                    </property>
                </widget>
            </item>
            <item row="0" column="1">
                <widget class="QLineEdit" name="lineEditOrderNumber"/>
            </item>
            <item row="1" column="0">
                <widget class="QLabel" name="labelCustomer">
                    <property name="text">
                        <string>Покупатель</string>
                    </property>
                </widget>
            </item>
            <item row="1" column="1">
                <widget class="QLineEdit" name="lineEditCustomer"/>
            </item>
            <item row="2" column="0">
                <widget class="QLabel" name="labelProduct">
                    <property name="text">
                        <string>Товар</string>
                    </property>
                </widget>
            </item>
            <item row="2" column="1">
                <widget class="QLineEdit" name="lineEditProduct"/>
            </item>
            <item row="3" column="0">
                <widget class="QCheckBox" name="checkBoxDiscount">
                    <property name="text">
                        <string>Скидка</string>
                    </property>
                </widget>
            </item>
            <item row="3" column="1">
                <widget class="QCheckBox" name="checkBoxShipping">
                    <property name="text">
                        <string>Доставка</string>
                    </property>
                </widget>
            </item>
            <item row="4" column="0" colspan="2">
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QPushButton" name="buttonOk">
                            <property name="text">
                                <string>OK</string>
                            </property>
                            <property name="minimumSize">
                                <size>
                                    <width>100</width>
                                    <height>0</height>
                                </size>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="buttonCancel">
                            <property name="text">
                                <string>Отмена</string>
                            </property>
                            <property name="minimumSize">
                                <size>
                                    <width>100</width>
                                    <height>0</height>
                                </size>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>